src/pages/index.js

```diff
@@ -79,16 +79,14 @@ imagePopup.setEventListeners();
 const deleteCardPopup = new PopupWithConfirmation("#card__delete-modal");
 deleteCardPopup.setEventListeners();
 
-const profileUpdateSubmit = new PopupWithConfirmation("#profile-edit-modal");
-const newCardSubmit = new PopupWithConfirmation("#add-new-card");
-
 function handleDeleteClick(card) {
   deleteCardPopup.open();
   deleteCardPopup.setSubmitAction(() => {
     deleteCardPopup.setSubmitText(true, "Deleting...");
     api
       .deleteCard(card.id)
       .then(() => {
+        deleteCardPopup.setSubmitText(true, "Deleted...");
```

Comment:
`        deleteCardPopup.setSubmitText(true, "Deleted...");
`

This is not needed here because `finally` will change the button text immediately and you will not see this text

Reply:
`        deleteCardPopup.setSubmitText(true, "Deleted...");
`

This is not needed here because `finally` will change the button text immediately and you will not see this text